<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget_COM">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>NET_CLIENT</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_Msg"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit_SendMsg"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Send">
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>清空显示</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>IP地址：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_IP">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>127.0.0.1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>端口：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Port">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>18888</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Connect">
            <property name="text">
             <string>连接</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>NET_SERVER</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_Server_Recv_Msg"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLineEdit" name="lineEdit_Server_Send_Msg"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Server_Send">
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>本机IP:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Server_IP">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>监听端口:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Server_ListenPort">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>1888</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Server_Listen">
            <property name="text">
             <string>开始监听</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_Server_Session">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>SERIAL</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_Ser_Msg"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>串口号</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_Ser_Num"/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>波特率</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_Ser_Baud">
            <property name="currentText">
             <string>4800</string>
            </property>
            <item>
             <property name="text">
              <string>4800</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>38400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>115200</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Ser_Open">
            <property name="text">
             <string>打开</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_Rcv_Hex">
            <property name="text">
             <string>HEX</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Ser_ClearRev">
            <property name="text">
             <string>清空显示</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLineEdit" name="lineEdit_Ser_Send"/>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_Send_Hex">
            <property name="text">
             <string>HEX</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_Send_NewLine">
            <property name="text">
             <string>换行</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Ser_Send">
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Chart</string>
       </attribute>
       <widget class="QCustomPlot" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>581</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
